embed_test.go

```diff
@@ -73,17 +73,17 @@ func TestEncodeDemo(t *testing.T) {
 		// Try to encode as binary
 		b, err := data.ToWire(tc.in)
 		if assert.Nil(err, "%d: %#v", i, tc.in) {
-			err := data.FromWire(b, tc.out)
-			if assert.Nil(err) {
+			err2 := data.FromWire(b, tc.out)
```

Comment:
this can be `err = ..`

Reply:
it shadows the previous declaration on L74

Reply:
so?)

Reply:
throws an error on `vetshadow`? should we not be trying to comply with the linting best practices?